modules/dataframe/include/inviwo/dataframe/io/json/dataframepropertyjsonconverter.h

```diff
@@ -31,7 +31,7 @@
 
 #include <inviwo/dataframe/dataframemoduledefine.h>  // for IVW_MODULE_DATAFRAME_API
 
-#include <nlohmann/json.hpp>  // for json
+#include <nlohmann/json.hpp>  // for ordered_json
```

Comment:
If you are changing this everywhere an we put this include and the new using line in it's own inviwo header, so we have it consistent everywhere... 
We don't want to end up pulling a headers with 
using json = nlohmann::json;
and 
using json = ::nlohmann::ordered_json;
at the same time....

Reply:
Good suggestion.